Formula/elektra.rb

```diff
@@ -0,0 +1,42 @@
+class Elektra < Formula
+  desc "Configuration Framework"
+  homepage "https://libelektra.org"
+  url "https://www.libelektra.org/ftp/elektra/releases/elektra-0.8.21.tar.gz"
+  sha256 "51892570f18d1667d0da4d0908a091e41b41c20db9835765677109a3d150cd26"
+  head "https://github.com/ElektraInitiative/libelektra.git"
+
+  option "with-qt", "Build GUI frontend"
+
+  depends_on "cmake" => :build
+  depends_on "doxygen" => :build
+  depends_on "qt" => :optional
+  depends_on "discount" if build.with? "qt"
+
+  def install
+    tools = ["kdb"]
+    tools << "qt-gui" if build.with? "qt"
+
+    args = std_cmake_args + %W[
+      -DBINDINGS='cpp'
+      -DTOOLS='#{tools.join ";"}'
+      -DPLUGINS='NODEP'
+    ]
+
+    mkdir "build" do
```

Comment:
does an in-tree build not work?

Reply:
It should work, but it is [not recommended](https://github.com/ElektraInitiative/libelektra/blob/master/scripts/configure-common#L29-L31).

Reply:
ok